content/docs/02-getting-started/00-choosing-a-provider.mdx

```diff
@@ -39,6 +39,12 @@ Add your API key to your environment:
 AI_GATEWAY_API_KEY=your_api_key_here
 ```
 
+<Note>
+  When deployed to Vercel, you can also use [OIDC
+  authentication](/providers/ai-sdk-providers/ai-gateway#oidc-authentication-vercel-deployments)
+  without API keys.
+</Note>
```

Comment:
I think that's not quite right.

As far as I understand it, the `oidc` token will be automatically retrieved/refreshed under the following circumstances.

1. The code is running on Vercel. (Vercel injects the `x-vercel-oidc-token'` request header which is then read out [here](https://github.com/vercel/vercel/blob/2fdee16711f0c87c7eb23f722d0b289a0d6ca5fc/packages/oidc/src/get-vercel-oidc-token.ts#L83C29-L83C49)
2. Local development. The use can use the `vc` CLI to connect the current folder to a Vercel project, which enables retrieval and refresh of OIDC tokens via [this code](https://github.com/vercel/vercel/blob/2fdee16711f0c87c7eb23f722d0b289a0d6ca5fc/packages/oidc/src/token.ts).